### PR TITLE
Add Outdoor Temperature Data for Each Activity

### PR DESCRIPTION
**Pull request by LuisAntonio5**
This PR adds two scripts and installs axios:
- One of the scripts, `add-outdoor_temp-col.ts`, adds the column 'outdoor_temp' to the activities table;
- The other script, `update-outdoor_temp.ts`, calculates the outdoor temperature for all activities and stores them in the activities table.

In order to test the scripts, make sure to first run `add-outdoor_temp-col.ts` if your activities table still doesn't have the 'outdoor_temp' column. (e.g: _$ bun add-outdoor_temp-col.ts_)

After your table has the 'outdoor_temp' column, you can run `update-outdoor_temp.ts`. (e.g: _$ bun update-outdoor_temp.ts_)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-        "typescript": "5.0.0"
-    },
-    "dependencies": {
-        "@types/prompts": "2.4.9",
-        "pg": "8.11.3",
-        "prettier": "3.2.4",
-        "prompts": "2.4.2",
-        "strava-v3": "2.2.0"
-    }
+  "name": "strava-pg",
```
Comment (davidgomes) [inline package.json]: Could you please run prettier on all your files? There's a `.prettierrc` file at the root of the repository to make sure that code styling is consistent.

File: add-outdoor_temp-col.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { Pool } from "pg";
```
Comment (davidgomes) [inline add-outdoor_temp-col.ts]: Can you rename this file to `add-outdoor-temp-col.ts` for consistency?

File: update-outdoor_temp.ts [hunk 3]
```diff
@@ -0,0 +1,133 @@
+import axios from "axios";
```
Comment (davidgomes) [inline update-outdoor_temp.ts]: Can you rename this file to `update-outdoor-temp.ts` for consistency?

**Review by davidgomes — CHANGES_REQUESTED** [review]
Can you update the README with some instructions on how to use this new script?

File: update-outdoor_temp.ts [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+            const endDate = new Date(startDate.getTime() + elapsed_time * 1000);
+            const daysAgo =
+                (new Date().getTime() - startDate.getTime()) /
+                (1000 * 60 * 60 * 24);
+
+            let apiEndpoint: string;
+            let params: any;
+
+            if (daysAgo > 3) {
+                apiEndpoint = "https://archive-api.open-meteo.com/v1/era5";
```
Comment (davidgomes) [inline update-outdoor_temp.ts]: So is this like a free API, no need for authentication of any sort?

File: package.json (hunk 1 above)
Comment (LuisAntonio5) [reply package.json]: Done!

File: add-outdoor_temp-col.ts (hunk 2 above)
Comment (LuisAntonio5) [reply add-outdoor_temp-col.ts]: Done!

File: update-outdoor_temp.ts (hunk 3 above)
Comment (LuisAntonio5) [reply update-outdoor_temp.ts]: Done!

File: update-outdoor_temp.ts (hunk 4 above)
Comment (LuisAntonio5) [reply update-outdoor_temp.ts]: Yup - it's free for non-commercial use. They don't restrict access, but ask for fair use and attribution. More info [here](https://open-meteo.com/en/pricing)

File: add-outdoor-temp-col.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+const pool = new Pool({
+    connectionString: process.env.PG_CONNECTION_STRING,
+});
+
+const client = await pool.connect();
+
+try {
+    await client.query(`
```
Comment (davidgomes) [inline add-outdoor-temp-col.ts]: It doens't seem like this file went through prettier, there's some tabs in here still

**Review by davidgomes — CHANGES_REQUESTED** [review]

File: add-outdoor-temp-col.ts (hunk 5 above)
Comment (LuisAntonio5) [reply add-outdoor-temp-col.ts]: Weird. I'm running: 
`npx prettier --write add-outdoor-temp-col.ts --config .prettierrc`
and getting `add-outdoor-temp-col.ts 285ms (unchanged)` 🤔

The only thing that got prettier to actually change the tabs into spaces was writing 
`client.query("ALTER TABLE activities ADD COLUMN IF NOT EXISTS outdoor_temp double precision;",);`

in the same line and then run prettier on the file... Might this be some from some conflicts between prettier and another formatter?

Comment (LuisAntonio5) [reply add-outdoor-temp-col.ts]: I've pushed the manual fix though

**Review by davidgomes — APPROVED** [review]

**Pull request merged**